### PR TITLE
reporter: add append option to tap and junit tests

### PR DESCRIPTION
**Pull request by GeorgeAdams95**
Fix for issue: #193

### CODE REVIEW

**Comment by GeorgeAdams95** [comment]
Fix for issue: https://github.com/nodejs/citgm/issues/193

**Comment by TheAlphaNerd** [comment]
@GeorgeAdams95 can you please add a test for the new feature.

Thanks!

File: lib/common-args.js [hunk 1]
```diff
@@ -36,6 +36,11 @@ module.exports = function commonArgs (app) {
     type: 'string',
     description: 'Path to prepend to $PATH when running tests'
   })
+  .option('append', {
+    alias: 'a',
+    type: 'boolean',
+    description: 'Turns on append to file mode'
```
Comment (TheAlphaNerd) [inline lib/common-args.js]: I'd like a slightly more in depth description here. Mentioning that it will append rather than replace.

Comment (GeorgeAdams95) [inline lib/common-args.js]: made those changes, i'll add a test tomorrow

**Comment by TheAlphaNerd** [comment]
Can you please include a test of both citgm and citgm-all that includes the append flag as an integration test.

Also, what happens if append is passed a path that does not exist yet? Should CITGM throw if the file doesn't exist? Should it simply create the file if it doesn't exist (this is the behavior of node)? We should likely have a test for that edge case to make sure that the behavior is expected.

**Comment by TheAlphaNerd** [comment]
LGTM landed in 2fe1da0b90

**Comment by TheAlphaNerd** [comment]
LGTM landed in 2fe1da0b90
